javascript/javascript1/week3/homework-js1-w3.js

```diff
@@ -0,0 +1,120 @@
+//**********************************************/
+// WARM UP
+//**********************************************/
+
+//--------------------------
+// Item array removal
+//--------------------------
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1); // cannot use slice() here because slice doesn't modify the original array
+  }
+}
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//--------------------------
+// When will we be there??
+//--------------------------
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function time(obj) {
+  const timeInPercentage = obj.destinationDistance / obj.speed;
+  const hour = Math.floor(timeInPercentage);
+  const minute = Math.floor((timeInPercentage - hour) * 60);
+  return `${hour} hours and ${minute} minutes`;
+}
+
+const travelTime = time(travelInformation);
+console.log(travelTime); // 8 hours and 38 minutes
```

Comment:
Really good work! Something which the exercise didn't ask for but I have often had to cater for in the past is also adding a conditional to check if it's a `minute` (singular) or `minutes` (plural) and based one the minute variable then add the "s" or not (same goes for hour(s))